Models/SSBTaxAssessment/POCO/LabeledValueTax.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections.ObjectModel;
+using System.Text.Json.Serialization;
+
+namespace Backend.Models.SSBTaxAssessment.POCO
+{
+    public class LabeledValueTax
+    {
+        [JsonPropertyName("municipality")] public string Municipality { get; init; }
+
+        /// Format: [ [ year, taxes ] ]
```

Comment:
I don't think the triple-slash documentation can be used for properties. 

Use double instead.

Reply:
Or correct me if I'm wrong!

Reply:
@Kpaubert will have to answer this, it's a copy of https://github.com/IT2901-SINTEF01/backend/blob/master/Models/SSBPopulationStatistics/POCO/LabeledValue.cs

Reply:
Apparently I wrote that line of code 😨 

I don't think it works, but it's not very important either so either leave it or change both. It's up to you:)

Reply:
Will just leave it there